README.md

```diff
@@ -27,7 +27,9 @@ To use the bridge you'll need:
 - **Windows 10**, build 10586 or higher. Validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system).
 - **Visual Studio 2017** with Windows developer tools. Visual Studio 2017 Community is available for free [here](https://dev.windows.com/downloads).
 
-   Ensure the following individual components are selected during installation (Selecting the **Universal Windows Platform development** workflow should give most of the below):
+  
+   <details><summary>Ensure the following individual components are selected during installation. Selecting the Universal Windows Platform development workflow should give most of the below (Click to Expand)</summary><p>
```

Comment:
>Click to Expand [](start = 191, length = 15)

nit: might make sense to make this look like a link; the 'click to expand' didn't really jump out at me at first